### DISCUSSION TITLE
New Source: AppTweak API

### DISCUSSION THREAD

**Discussion by vishalhtg** [airbytehq/airbyte#36103] [New Connector Request]
## Tell us about the new connector you’d like to have
* AppTweak data to different DWH (i.e. Snowflake)
* API Documentation: https://www.apptweak.io/documentation
* Main Website: https://www.apptweak.io/

## Describe the context around this new connector
* Need to fetch data for app ranking, reviews and competitors from Apple store and Google Play store

## Describe the alternative you are considering or using
Alternate option write an API on python and deploy it on our server to push data on S3 bucket from where we can easily move data to DWH.

## Are you willing to submit a PR?
Yes, I can contribute on this connector 🙌🏼

**Comment by marcosmarxm** [comment]
Let me know if you need any assistance to build the connector @vishalhtg

**Comment by arnaudselva** [comment]
+1 for this connector 🙏  any plan to work on this @vishalhtg ?

[a comment by an automated account is not shown]
